_pages/WaitMs.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: WaitMs routine
+permalink: /wait_ms/
```

Comment:
I think we should be consistent with the URLs and seperate with hyphen instead of underscore

Reply:
fixed